modules/gdscript/tests/scripts/analyzer/warnings/shadowing_base.notest.gd

```diff
@@ -0,0 +1,7 @@
+class_name ShadowingBase
+
+const base_const_member = 1
```

Comment:
Looks unused.

Reply:
I added a test cast for in the `shadowning.gd`.